unit/memory-analyzer/gdb_api.cpp

```diff
@@ -104,94 +104,111 @@ void gdb_api_internals_test()
 TEST_CASE("gdb api internals test", "[core][memory-analyzer]")
 {
 #ifdef RUN_GDB_API_TESTS
-  gdb_api_internals_test();
+  const bool has_gdb = system("gdb --version") == 0;
```

Comment:
I would prefer if `run("gdb", {"gdb", "--version"})` (from `util/run.{h,cpp}`) were used in place of `system` (`system` is really heavy-weight as it starts a shell). But I do acknowledge that we already use `system` in this file, though I think that call should equally be replaced.

Reply:
Replaced. The other use was replaced in the next commit.